src/globus_cli/commands/endpoint/user_credential/create/posix.py

```diff
@@ -0,0 +1,37 @@
+from globus_sdk.services.gcs import UserCredentialDocument
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command, endpoint_id_arg
+from globus_cli.termio import formatted_print
+
+from .._common import user_credential_create_and_update_params
+
+
+@command("posix", short_help="Create a User Credential for a POSIX storage gateway")
+@endpoint_id_arg
+@user_credential_create_and_update_params(create=True)
+@LoginManager.requires_login(LoginManager.TRANSFER_RS, LoginManager.AUTH_RS)
+def posix(
+    *,
+    login_manager: LoginManager,
+    endpoint_id,
+    storage_gateway,
+    globus_identity,
+    local_username,
+    **kwargs,  # values from user_credential_create_and_update_params
```

Comment:
How much would it take to eliminate `**kwargs` usage here?
I've been trying to remove/avoid `**kwargs` for our commands because it doesn't play nicely with type checking.

In this case, I think we have `display_name: str | None` and nothing else to add.

Reply:
Just to note: if you take that suggestion and use `str | None`, make sure to add `from __future__ import annotations`.